INSTALL.md

```diff
@@ -1,16 +1,14 @@
 # Installation
 
-**Note**: For Windows and macOS, installation is not required to use Porymap. You can download the latest release to begin using Porymap immediately.
+For installation, Porymap requires tools for Qt 5.14.2 & C++11 or newer.
```

Comment:
Please restore this line to its original wording and position (people coming from the pret repos are used to heading straight to `INSTALL`, so it's important for the first thing they see in here to be the "you don't need to do this" note, and Qt itself is required, not tools for Qt).